tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt

```diff
@@ -968,3 +968,5 @@ wikimedia ; * ; moderate ;
 
 yahoo ; mul ; modern ;
 # ^no votes in 30…40 for this org
+
+iaap ; @weight=4 @pathGroup=characters_and_annotations * ; moderate ;
```

Comment:
I put the @ in the locale section, because it is a modifier on the allowed locales.